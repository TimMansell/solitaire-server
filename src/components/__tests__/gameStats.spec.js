import { shallowMount } from '@vue/test-utils';
import GameStats from '@/components/GameStats.vue';

describe('GameStats.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(GameStats, {
      computed: {
        timer: () => 1,
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
