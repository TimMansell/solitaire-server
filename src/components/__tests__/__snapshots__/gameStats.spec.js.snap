// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameStats.vue matches snapshot 1`] = `
<div class="stats">
  <div class="stats__item">
    <usergames-stub></usergames-stub>
  </div>
  <div class="stats__item">
    <moves-stub></moves-stub>
  </div>
  <div class="stats__item">
    <timer-stub></timer-stub>
  </div>
</div>
`;
